events/event_handler.go

```diff
@@ -160,6 +161,18 @@ func (fh *eventHandler) handleStatusEvent(body []byte) error {
 		"TaskStatus": task.TaskStatus,
 	}).Info("Got StatusEvent")
 
+	app, err := fh.marathon.App(task.AppID)
```

Comment:
Marathon-consul sets different service ID then mesos-consul so it won't delete apps registered by mesos-consul.